include/zephyr/net/socket.h

```diff
@@ -41,6 +41,10 @@ struct zsock_pollfd {
 	short revents;
 };
 
+/**
+ * @defgroup poll_options Options for poll()
```

Comment:
I think you might be looking at using `@name` groups, rather than `@defgroup`. The former is more geared towards semantically grouping stuff together, whereas the latter ends up calling the group "modules", which is probably more than what you need? Added a task to #6810 so that I document this as a guideline at some point :)

Reply:
Thanks for the hints! I will change to use `@name` groups then.